### PR TITLE
Add Check in views

### PR DESCRIPTION
**Pull request by casassg**
Check #21 for details on this

### CODE REVIEW

File: app/settings.py [hunk 1]
```diff
@@ -24,7 +24,8 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = ['localhost', 'hackupc.com', 'my.hackupc.com']
+
+ALLOWED_HOSTS = ['localhost', 'hackupc.com', 'my.hackupc.com', '127.0.0.1']
```
Comment (rgascons) [inline app/settings.py:28]: what's the reason of adding 127.0.0.1 as a host?

**Comment by rgascons** [comment]
- Caps in all @casassg commits
- What exactly commit 2dd227e is? Is this a merge commit, or a rebase one? Either way shouldn't be there. If it's too much hassle leave it as is, but take it into account for future PRs

**Comment by zakum42** [comment]
What exactly commit 2dd227e is? Is this a merge commit, or a rebase one? Either way shouldn't be there. If it's too much hassle leave it as is, but take it into account for future PRs

**Comment by zakum42** [comment]
Also, do all commits build?

File: app/settings.py (hunk 1 above)
Comment (casassg) [reply app/settings.py:28]: I guess @alvarogirona wanted to use it as host to try production mode? Idk, didn't add it myself I always use localhost

**Comment by casassg** [comment]
@zakum42 

- I've always been using non-caps (easier, faster and same result). If you tell me why I should switch I will, it's just a habit I have had for a while, and nobody complained.
-  This commit was me fixing stuff and suddenly realizing that I did too many stuff. It's not a merge or rebase, it's a refactor commit (as it's name says). Therefore it has a lot of changes. I should have done it separately. It includes bug fixes (code was not compiling correctly before) + style checks (follow PEP08 pls) + refactor of views code (improve readability) + move templates to new folder (for some reason github is saying that I deleted some code and added new files)

**Comment by rgascons** [comment]
@casassg 

- Caps or no-caps, not both :)
- k

**Comment by casassg** [comment]
I always go by no-caps. Also mark as approved if good pls :D

**Pull request merged**